lib/ts/ink_queue.cc

```diff
@@ -52,6 +52,11 @@
 #include "ts/hugepages.h"
 #include "ts/Diags.h"
 
+
+// in order to compile correctly
```

Comment:
I'm not commenting on the number of lines here...

I am confused by this warning comment. It is telling me that we have to undef `HAVE_LIBJEMALLOC`, but doesn't indicate why, in enough detail for me to know what danger I'm avoiding.

Can you expound upon what "compile correctly" means in this context? Why must we undefine that symbol?